code/script.js

```diff
@@ -1,9 +1,14 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
-}
+// Find all elements with the class "accordion-item-header" and assign them to the variable "accordionItemHeader".
+const accordionItemHeaders = document.querySelectorAll(".accordion-item-header");
 
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+// Put all accordionItemHeader in the accordionItemHeaders ListeningStateChangedEvent, and toggle the class active depending on whether they already are active or not 
+accordionItemHeaders.forEach(accordionItemHeader => {
+  accordionItemHeader.addEventListener("click", event => {
+    accordionItemHeader.classList.toggle("active");
+  });
+});
+
+// This one updates the slider range. Couldn't get it to update on change, unfortunately.
+function updateTextInput(val) {
```

Comment:
Not sure if you are aware, but over here you covered advanced topic we will talk about during React weeks called "Controlled inputs". Great job :)
On the other hand, once again, are you aware that your input is going to completely fine without this function?